spring-boot/pom.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>infinispan-simple-tutorials-spring-boot</artifactId>
+    <parent>
+        <relativePath>../pom.xml</relativePath>
+        <version>1.0.0-SNAPSHOT</version>
+        <groupId>org.infinispan.tutorial.simple</groupId>
+        <artifactId>infinispan-simple-tutorials</artifactId>
+    </parent>
+
+    <name>Infinispan Simple Tutorials: Spring Boot</name>
+
+    <properties>
+        <spring-boot.version>1.4.0.RELEASE</spring-boot.version>
+        <infinispan.starters.version>1.0.0-SNAPSHOT</infinispan.starters.version>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-starter-parent</artifactId>
+                <version>${spring-boot.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.infinispan</groupId>
+            <artifactId>infinispan-remote</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.infinispan</groupId>
+            <artifactId>inifinispan-spring-boot-starter</artifactId>
+            <version>${infinispan.starters.version}</version>
```

Comment:
IMO your sample should expose how the project will be configured in the target environnement. No promise here and even though we've never done this before, we might provide dependency management for this starter in Spring Boot itself, since we provide dependency management for Infinispan.